### PR TITLE
feature: add scale option

### PR DESCRIPTION
**Pull request by DemianD**
This PR adds an option to set the scale of a PDF page.

https://pptr.dev/#?product=Puppeteer&version=v5.5.0&show=api-class-page

image

**Checklist**

- [x] PSR-2 Coding Standard - The easiest way to apply the conventions is to install PHP Code Sniffer.

- [x] Add tests! - Your patch won't be accepted if it doesn't have tests.

- [x] Document any change in behaviour - Make sure the README.md and any other relevant documentation are kept up-to-date.

- [x] Consider our release cycle - We try to follow SemVer v2.0.0. Randomly breaking public APIs is not an option.

- [x] Create feature branches - Don't ask us to pull from your master branch.

- [x] One pull request per feature - If you want to do more than one thing, send multiple pull requests.

- [x] Send coherent history - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please squash them before submitting.

### CODE REVIEW

File: src/Browsershot.php [hunk 1]
```diff
@@ -240,6 +240,7 @@ public function clip(int $x, int $y, int $width, int $height)
     public function select($selector, $index = 0)
     {
         $this->selectorIndex($index);
+
```
Comment (DemianD) [inline src/Browsershot.php:243]: This is an unrelated change to this PR, but the CI is failing on this line.

**Comment by DemianD** [comment]
Hey @freekmurze

I hope you are well. 
Thank you for creating this package (and all other packages of course!).

I'm wondering if you'd have some time to review this pull request.
If I can help with anything, or make it easier for you to test, let me know.

**Comment by freekmurze** [comment]
Thanks!

**Comment by vinnygambiny** [comment]
They are a way to auto-scale? Like, having a large table and I want to fit on a Letter page size?

**Pull request merged**